metrics/web/test/common/presentation/metrics_theme/model/build_results_theme_data_test.dart

```diff
@@ -1,40 +1,57 @@
-// https://github.com/software-platform/monorepo/issues/140
-// ignore_for_file: prefer_const_constructors
-import 'package:flutter/cupertino.dart';
+import 'package:metrics/common/presentation/colored_bar/theme/style/metrics_colored_bar_style.dart';
 import 'package:metrics/common/presentation/metrics_theme/config/color_config.dart';
-import 'package:metrics/common/presentation/metrics_theme/model/build_results_theme_data.dart';
+import 'package:metrics/common/presentation/metrics_theme/model/build_result_bar/theme/attention_level/build_result_bar_attention_level.dart';
+import 'package:metrics/common/presentation/metrics_theme/model/build_result_bar/theme/theme_data/build_result_bar_theme_data.dart';
 import 'package:test/test.dart';
 
+// https://github.com/software-platform/monorepo/issues/140
+// ignore_for_file: prefer_const_constructors
+
 void main() {
   group("BuildResultsThemeData", () {
     test(
       "creates a new instance with given colors and textStyle",
       () {
-        const textStyle = TextStyle(
-          fontSize: 12.0,
-          fontWeight: FontWeight.bold,
+        final buildResultsThemeData = BuildResultBarThemeData(
```

Comment:
It seems like here we should test only that the BuildResultsThemeData creates a new instance with the given `attentionLevel` and that it creates a new instance with the default attention level if the given one is null.